Makefile tries to vendor under /_vendor on OpenBSD/amd64

with GNU make 
```
fishtank$ gmake --version
GNU Make 4.2.1
Built for x86_64-unknown-openbsd6.2
Copyright (C) 1988-2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
fishtank$ echo $GOPATH
/home/mulander/junk/notes/go
fishtank$ export GOPATH
fishtank$ pwd
/home/mulander/junk/notes/go/src/github.com/evilsocket/arc/arcd
fishtank$ gmake vendor_get 
rm -dRf _vendor/src
GOPATH=/_vendor go get -d -u -v github.com/gin-gonic/gin github.com/theckman/go-flock gopkg.in/gomail.v2 github.com/stretchr/testify/assert github.com/dgrijalva/jwt-go 
github.com/gin-gonic/gin (download)
package github.com/gin-gonic/gin: mkdir /_vendor: permission denied
github.com/theckman/go-flock (download)
package github.com/theckman/go-flock: mkdir /_vendor: permission denied
Fetching https://gopkg.in/gomail.v2?go-get=1
Parsing meta tags from https://gopkg.in/gomail.v2?go-get=1 (status code 200)
get "gopkg.in/gomail.v2": found meta tag get.metaImport{Prefix:"gopkg.in/gomail.v2", VCS:"git", RepoRoot:"https://gopkg.in/gomail.v2"} at https://gopkg.in/gomail.v2?go-get=1
gopkg.in/gomail.v2 (download)
package gopkg.in/gomail.v2: mkdir /_vendor: permission denied
github.com/stretchr/testify (download)
package github.com/stretchr/testify/assert: mkdir /_vendor: permission denied
github.com/dgrijalva/jwt-go (download)
package github.com/dgrijalva/jwt-go: mkdir /_vendor: permission denied
gmake: *** [Makefile:58: vendor_get] Error 1
fishtank$ echo $?
2
```

with BSD make
```
fishtank$ make
*** Parse error in /home/mulander/junk/notes/go/src/github.com/evilsocket/arc/arcd: Need an operator in 'GOPATH' (Makefile:17)
fishtank$
```

--- a/arcd/Makefile
+++ b/arcd/Makefile
@@ -13,7 +13,7 @@ TARGET=arcd
 # Prepend our _vendor directory to the system GOPATH
 # so that import path resolution will prioritize
 # our third party snapshots.
-GOPATH := ${PWD}/$(VENDOR_PATH):${GOPATH}
+GOPATH := ${CURDIR}/$(VENDOR_PATH):${GOPATH}
 export GOPATH
 
 default: build
@@ -55,7 +55,7 @@ vendor_update: vendor_get
 # installed in GOPATH since `go get` will use
 # that existing location as the destination.
 vendor_get: vendor_clean
-	GOPATH=${PWD}/$(VENDOR_PATH) go get -d -u -v $(DEPENDENCIES)
+	GOPATH=${CURDIR}/$(VENDOR_PATH) go get -d -u -v $(DEPENDENCIES)
 
 vendor_clean:
 	rm -dRf $(VENDOR_PATH)/src

